src/Game.cpp

```diff
@@ -196,6 +374,27 @@ void sub_6134D0(const char *var8) {
     }
 }
 
+bool sub_612EB0(const char * key, std::uint32_t value_index, const char ** value_arg) {
+    unk_612150 *value = glo_8600F8.sub_6124A0(key);
+
+    if (value == nullptr)
+        return false;
+
+    if (value_index >= value->count)
+        return false;
```

Comment:
Note to self: `if (value == nullptr || value->count <= value_index)` would be more concise if I ever decide to clean this up.